x-pack/legacy/plugins/index_management/public/sections/template_create/template_create.tsx

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
For the clone functionality, instead of having all the if / else statement I would have created a separate component. Don't you think it would be cleaner?

Reply:
That's a valid point. I had considered that originally, but now that you bring it up again, I think that is the better approach. I went ahead and broke this up into a separate route and component.

Reply:
Great, thanks!